docs/maintainers/continuous-integration.md

```diff
@@ -56,10 +56,10 @@ You can install the Concept CI checks for your track by adding your track to the
 Add the following below the existing paths in the `on: push: paths:` and `on: pull_request: paths` keys:
 
 ```yaml
-    - 'languages/<language>/config.json'
-    - 'languages/<language>/reference/concepts.csv'
-    - 'languages/<language>/reference/exercise-concepts/**'
-    - 'languages/<language>/exercises/concept/**'
+- "languages/<language>/config.json"
+- "languages/<language>/reference/concepts.csv"
+- "languages/<language>/reference/exercise-concepts/**"
+- "languages/<language>/exercises/concept/**"
 ```
```

Comment:
YAML whitespace is significant; removing the leading spaces here likely makes it harder to copy/paste this data into the target document.

Reply:
Indeed, this whitespace should not be removed.

Reply:
Good catch. I've updated